docs/readmes/contributing/contribute_conventions.md

```diff
@@ -281,6 +282,31 @@ The [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html)
     - Default to `.bash` except with specific reason
 - When a shell script passes around 100 lines, it's time to re-write it in Python or Go
 
+### Javascript
+
+**General**
+- Importing order should be: types, default components, and named components with each separated by a newline
+- Use `CamelCase` for component file names
+- Favour `map` and `forEach` functions in place of regular `for` loops
+- Refrain from using literal strings/numbers without defining them
+- Use strict equality(`===`) when comparing values
+
+**Type Annotations**
```

Comment:
I think for **Type Annotations** and **Linter** we might want to add a line to link the dev_testing instructions.

Reply:
Yeah, that was what I think of too. So, I linked flow and eslint official website for now but will change them to link to dev_testing instruction when that pull request gets approved cause that makes more sense.

Reply:
```suggestion
**Type annotations**
```

Reply:
@themarwhal I have now added the link to the dev_testing instructions since that got merged.